audit_scope.md

```diff
@@ -0,0 +1,32 @@
+## VVV Venture Capital Audit Scope
+
+1. **Scope** 
+	1. Venture Capital Investment and Claims (`contracts/vc/`)
+		1.  `VVVVCInvestmentLedger.sol`
+			1. Facilitates user investments in vVv's venture capital projects
+		2.  `VVVVCTokenDistributor.sol`
+			1. Facilitates users' token claims for vested tokens, based on prior investment
+	3. nSLOC
+		1. in `contract_metrics_for_audit.md`
+2. **Smart Contract Requirements**
+	1. Venture Capital Investment and Claims (`contracts/vc/`)
+		1. `VVVVCInvestmentLedger.sol`
+			1. Admin
+				1. All admin-only functions are gated via the `onlyAuthorized` modifier from `contracts/auth/VVVAuthorizationRegistryChecker.sol`.
+				2. Can pause all investing via this contract
+				3. Can manually add a record of investment contribution to an investment round
+				4. Can manually issue a refund to a specific user via this contract
+			2. User
+				1. Can invest in an active investment round **only** when `investmentIsPaused` is `false`, within the contribution limit set in the signature supplied to the `invest` call, within the start and end times of the investment round, and when the input signature is valid
+				2. An admin-set fee (signature parameter defined off-chain) is only taken from the user-invested amount if the fee value is positive, and when a fee is taken, the correct amount is taken.
+				3. A random user cannot access any functions gated via the `onlyAuthorized` modifier.
```

Comment:
Got it, adapted the terminology here too to  more clearly specify things as "role-restricted" rather than "admin-only". The only mention of "admin" is now with respect to the off-chain setting of a fee for each round.